### PR TITLE
Adding general contribution guidance to tutorial.md

### PR DESCRIPTION
**Pull request by zpopp**
Added at the top of this tutorial a description of parameters for what makes appropriate and inappropriate contributions to the CAFE Dataverse Collection. Removed one line from the Tutorial about tabular data since it is included in the general guidelines.

### CODE REVIEW

File: handbook/tutorial.md [hunk 1]
```diff
@@ -1,5 +1,26 @@
...
+
+We strongly encourage the community of practice to contribute to the expansion of the CAFE Dataverse Collection. Emphasizing open access and collaborative research, the CAFÉ Collection invites contributions from a diverse array of stakeholders, including government agencies, NGOs, community-based organizations, industry partners, and academics. 
+
+Parameters for what datasets are appropriate and inappropriate for the CAFE Dataverse are described below:
+
+## General Guidance
+
+- Contributions should be relevant to climate and health research.
+- Contributions should not be identical to data stored in other repositories. The submission of processed derivatives or expansions of data accessible through existing sharing resources (ie: SEDAC, Google Earth Engine) are encouraged.
+- Contributions should be in line with the licensing of raw source data.
```
Comment (daniellebraun) [inline handbook/tutorial.md:13]: maybe remove "raw"?

File: handbook/tutorial.md [hunk 2]
```diff
@@ -1,5 +1,26 @@
...
+- Data contributors should only post data that they own, have generated, or have been granted permission to reshare in a manipulated version (ie: census data).
+- No restricted access data (ie: data including personal identifying information) should be shared through the CAFE Dataverse. Contributions will be widely accessible to Harvard Dataverse users.
```
Comment (daniellebraun) [inline handbook/tutorial.md:15]: maybe "restricted or regulated"?

**Review by daniellebraun — COMMENTED** [review]
@zpopp left two suggestions in my review, let me know what you think and i can then approve

File: handbook/tutorial.md (hunk 1 above)
Comment (zpopp) [reply handbook/tutorial.md:13]: Sounds good!

File: handbook/tutorial.md (hunk 2 above)
Comment (zpopp) [reply handbook/tutorial.md:15]: Restricted access/restricted use data has a specific definition around identifiability which I think is useful. Are there specific aspects of 'regulated' that we could add in parentheses to be clear about what we mean?

**Review by daniellebraun — APPROVED** [review]
looks good

**Pull request merged**